<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ui_PolylineOptions</class>
 <widget class="QWidget" name="Ui_PolylineOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>22</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>22</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>32</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Polyline Options</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="bClose">
     <property name="toolTip">
      <string>Connects ends of the polyline so closed contour is created</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="bUndo">
     <property name="toolTip">
      <string>Undo for previous vertex</string>
     </property>
     <property name="text">
      <string>Undo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Segment Type:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbMode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Defines which segment of polyline should be created</string>
     </property>
     <item>
      <property name="text">
       <string>Line</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tangential</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tan Radius</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Angle</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lRadius">
     <property name="text">
      <string>Radius:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leRadius">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Radius of arc</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lAngle">
     <property name="text">
      <string>Angle:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leAngle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Angle of arc</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="buttonGroup1">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbNeg">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Clockwise</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/extui/extui.qrc">
          <normaloff>:/extui/dirneg.png</normaloff>:/extui/dirneg.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbPos">
        <property name="toolTip">
         <string>Counter Clockwise</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/extui/extui.qrc">
          <normaloff>:/extui/dirpos.png</normaloff>:/extui/dirpos.png</iconset>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../../res/extui/extui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bClose</sender>
   <signal>clicked()</signal>
   <receiver>Ui_PolylineOptions</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bUndo</sender>
   <signal>clicked()</signal>
   <receiver>Ui_PolylineOptions</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbMode</sender>
   <signal>activated(int)</signal>
   <receiver>Ui_PolylineOptions</receiver>
   <slot>setModeToActionAndView(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>undo()</slot>
  <slot>setRadiusToActionAndView(QString)</slot>
  <slot>updateAngle(QString)</slot>
  <slot>setModeToActionAndView(int)</slot>
  <slot>updateDirection(bool)</slot>
 </slots>
</ui>
